contrib/IECoreUSD/include/IECoreUSD/AttributeAlgo.h

```diff
@@ -55,6 +55,8 @@ IECOREUSD_API pxr::TfToken cortexPrimitiveVariableMetadataToken();
 IECOREUSD_API pxr::TfToken toUSD( const std::string& name );
 IECOREUSD_API IECore::InternedString fromUSD( const std::string& name );
 IECOREUSD_API IECore::InternedString primVarPrefix();
+IECOREUSD_API std::string userPrefix();
+IECOREUSD_API std::string renderPrefix();
```

Comment:
For me, the idea behind `AttributeAlgo` was to completely abstract away all the details of what prefixes there are and what they mean. The idea was that everything would rely purely on `toUSD()` and `fromUSD()`, and that this would let us extend the logic without needing to update any client code. For this to be watertight, we'd also want to move `isCortexAttribute()` from `USDScene.cpp` and into `AttributeAlgo`.